docs/index.md

```diff
@@ -44,11 +44,42 @@ Configuration File:
 
 ```hcl
 connection "vault" {
-  plugin  = "theapsgroup/vault"
-  address = "https://vault.mycorp.com/"
-  token   = "s.f7Ea3C3ojOYE0GRLzmhSGNkE"
+  plugin    = "theapsgroup/vault"
+  address   = "https://vault.mycorp.com/"
+  auth_type = "token"
```

Comment:
Maybe add this extra configuration option to `config/vault.spc` with a descriptive comment (allowed values, etc)